src/atoms/Text/Text.native.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components/native';
+import theme from '../../tokens/theme';
+import { Text as NativeText } from 'react-native';
+import { getAllColorNames, resolveColorFromString } from '../../utils/colors';
+import automation from '../../_helpers/automation-attributes';
+
+const fontFamilies = {
```

Comment:
put this in `fonts.js`?

Reply:
This will only be used here. Do we still need to put it there?